sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Changed `FormRecognizerCommonClientOptions` to extend `CommonClientOptions` from `@azure/core-client` instead of `PipelineOptions`. `CommonClientOptions` itself extends `PipelineOptions`, so no fields are removed, but `CommonClientOptions` also includes `httpClient` and `allowInsecureConnection` fields to allow overriding the default HTTP client and using insecure connections (without SSL/TLS) respectively.
```

Comment:
Likely happened when migrating from core-http. Our template has this https://github.com/Azure/azure-sdk-for-js/blob/98279fc889ca386525ccc17844eea2387cb40c7c/sdk/template/template/src/configurationClient.ts#L41

Reply:
Ok, so we should be good to avoid future occurrences once the template is updated? If so https://github.com/Azure/azure-sdk-for-js/issues/18044 should cover it. cc @JonathanCrd who will be working on that issue soon

Should we also update the [wiki](https://github.com/Azure/autorest.typescript/wiki/%60core-http%60-dependency-migration-to-%60core-client%60-%60core-rest-pipeline%60) then?

Reply:
Yeah this is mostly just down to the old pattern being to extend `PipelineOptions`. The wiki page for migrating to corev2 actually does mention this, but this package wasn't a migration from core-http to core-rest-pipeline. It was a full rewrite.